zgrab_schema.py

```diff
@@ -820,15 +820,15 @@
                         }),
                         "serial":String(),
                         "cert_type":SubRecord({
-                            "id":Integer(),
+                            "id":Unsigned32BitInteger(),
```

Comment:
I'm not quite sure what it is, either, but it's populated from [this `uint32` field](https://github.com/zmap/zgrab/blob/42a47296c0227ba36e9a69ff57e6a585a5b81e4b/ztools/xssh/certs.go#L56), so I think this is an appropriate choice in the schema.